flocker/node/agents/blockdevice.py

```diff
@@ -55,6 +58,45 @@
 PROFILE_METADATA_KEY = u"clusterhq:flocker:profile"
 
 
+class DatasetStates(Names):
+    NON_MANIFEST = NamedConstant()
+    ATTACHED = NamedConstant()
+    MOUNTED = NamedConstant()
+    DELETED = NamedConstant()
+
+
+class DiscoveredDataset(PClass):
```

Comment:
Do you still intend on fixing these up before merging? Just pointing it out.

I think at least documenting the constants above makes sense.